### ISSUE TITLE
Add theme switch button for light/dark mode

### ISSUE THREAD

**Issue by batradilshan18** [anishmanchanda/DockWidgets#7]
Hi, I am a GSSoC'25 Contributor.
The website does not have a theme switch button to toggle between light and dark modes. I will add this functionality for better accessibility.
I would like to work on this alone.
Kindly assign this issue to me.
@anishmanchanda

**Comment by Anvesha-24** [comment]
hey i would like to work on this issue, Could you please assign it to me?

**Comment by Anvesha-24** [comment]
Hi, I realized that I don’t have access to a Mac to properly test this macOS-specific issue, so I won’t be able to work on it. Apologies for the inconvenience.

**Comment by batradilshan18** [comment]
@anishmanchanda please unassign this issue from Anvesh-24 as they said themselves that they won't be able to work on this.
